apps/server/src/stores/token-player-map.ts

```diff
@@ -31,4 +31,43 @@ export class TokenPlayerMap {
       this.map.set(authToken, { ...data, socketId });
     }
   }
+
+  //get player id by socket id
```

Comment:
Let's use jsdoc type comments to describe functions

Reply:
Docs updated 7c8500527f286823f1793b8cdd7cdb83b60b7268